rmgpy/rmg/model.py

```diff
@@ -1613,6 +1617,9 @@ def add_seed_mechanism_to_core(self, seed_mechanism, react=False):
         for spec in self.new_species_list:
             if spec.reactive:
                 submit(spec, self.solvent_name)
+            if self.liquid_volumetric_mass_transfer_coefficient_power_law:
```

Comment:
Why does this need done both in species processing and in reaction processing?

Reply:
You're right, it's not necessary! I tried to be consistent with where we call `submit` but it could be redundant